### ISSUE TITLE
Option to change calendar for added events missing

### ISSUE THREAD

**Issue by davidhedlund** [Etar-Group/Etar-Calendar#101]
In both S Planner and Etar you can choose what Calendar an Event should use, but only in S Planner it is possible to modify the Calendar once it have been added.

**Comment by davidhedlund** [comment]
https://github.com/Free-Software-for-Android/Standalone-Calendar/issues/14

**Comment by languitar** [comment]
I also desperately miss this. Once you have accidentally created an event in the wrong calendar, there is no hope without other software.

**Comment by amunizp** [comment]
I was able to change one appointment in tablet but did not get tge option on mobile. Strange.

**Comment by igoralmeida** [comment]
@Gitsaibot if you have given some thought to this (and, for that matter, every other 'help wanted' issue), it would be great if you wrote a bit how you would approach the problem/enhancement. It gives potential contributors important pointers and insight into the organization of the code, etc.

There is no need for structured thoughts or even well-written text, sometimes filenames alone are already a great starting point.

**Comment by gerhardbeck** [comment]
would love that, too. In Google Calendar I can copy the event into another calendar.
Better is the was of simple Calendar: I have a button to edith which calendar the event is assigned to

**Comment by Gitsaibot** [comment]
@igoralmeida Sorry for the late reply, but I forgot this problem entirely. To be honest, I have not thought much about the problem. There has to be a reason why Google took years to create a way to copy appointments. I think it has a lot to do with syncronization of the appointments and all the database stuff. It should also be noted that there is a difference between local and online synced calendars. I am not sure if there is a simple solution for this, but if so, then it lies beyond my possibilities :(. 

A start point could be EditEventView.java Line 979 or 1104

**Comment by rauferd** [comment]
I read this only after posting a comment to #87 (Copy Events), but I would like to add that copying events, as suggested in #87 may be much easier to implement and would probably solve the problem for most users:
They could use the (future) copy function, make the desired adjustments and then delete the old/incorrect event.

Just creating a new event and filling it with the data of an exisiting event should be much easier to do than _modifying_ the exisiting event consistently. The only downside that I can see is that the event id will change (which is irrelevant for most users?).

**Comment by alexanderadam** [comment]
Isn't this a duplicate of #62?

**Comment by PH-G** [comment]
Hello, I would really appreciate this feature as well. Maybe I'm thinking too simple, but couldn't you just export the Event to a temp file or a buffer, delete it in the source calendar, and then import it into the target calendar? This would move he event to the new calendar. You could even have an option to copy the event, in this case you simple leave out the delete step above. Would this be possible?

**Comment by jspricke** [comment]
It is pretty clear how to do it, someone needs to find the time to submit a patch.

**Comment by Gitsaibot** [comment]
We now have an open pull request that enables the desired function. It would help us if many could test this in advance.
